### PR TITLE
0.25.3 whatsnew

### PR DESCRIPTION
**Pull request by WillAyd**
- [ ] closes #xxxx
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

**Comment by jreback** [comment]
did we push the rolling patch to the 0.25.x branch?

**Comment by WillAyd** [comment]
No I was thinking merge this to master than I can backport this with the fix, though if you prefer another way lmk

**Comment by jreback** [comment]
no that’s fine

didn’t know if u did the backport yet

File: doc/source/whatsnew/v0.25.3.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+Groupby/resample/rolling
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Bug in :meth:`DataFrameGroupBy.quantile` where NA values in the grouping could cause segfaults or incorrect results (:issue:`28882`)
+
+Contributors
+~~~~~~~~~~~~
+
+.. contributors:: v0.25.2..v0.25.3
```
Comment (WillAyd) [inline doc/source/whatsnew/v0.25.3.rst]: @jgehrcke @TomAugspurger would either of you happen to know how to get this to work? Do I need to pin this to HEAD before making the release then update the whatsnew after tagging 0.25.3?

**Comment by TomAugspurger** [comment]
You need the v0.25.3 tag for that to work. Using HEAD is the best we can do
till the release.

**Pull request merged**